tests/kernel/install_ltp.pm

```diff
@@ -265,12 +265,11 @@ sub run {
     }
 
     # poo#18980
-    if (get_var('OFW') && check_var('VIRTIO_CONSOLE', 1)) {
+    if (get_var('OFW') && !check_var('VIRTIO_CONSOLE', 0)) {
```

Comment:
again looks like leftover from initial idea when we had NO_VIRTIO_CONSOLE , so let's just keep initial code since it is the same